apisix/cli/ops.lua

```diff
@@ -461,45 +461,238 @@ Please modify "admin_key" in conf/config.yaml .
         end
     end
 
-    -- support multiple ports listen, compatible with the original style
+    local ip_port_to_check = {}
+
+    -- support multiple ports listen, support specific IP, compatible with the original style
     if type(yaml_conf.apisix.node_listen) == "number" then
 
         if ports_to_check[yaml_conf.apisix.node_listen] ~= nil then
             util.die("node_listen port ", yaml_conf.apisix.node_listen,
                     " conflicts with ", ports_to_check[yaml_conf.apisix.node_listen], "\n")
         end
 
-        local node_listen = {{port = yaml_conf.apisix.node_listen}}
+        local node_listen = {}
+        table_insert(node_listen, {ip = "0.0.0.0", port = yaml_conf.apisix.node_listen})
```

Comment:
Strictly speaking, the default value of "ip" is "*", `listen 80` is equal to `listen *:80`.